### PR TITLE
Add Keras for NLP tutorial

### PR DESCRIPTION
**Pull request by nelson-liu**
This is a quick tutorial / talk to introduce Keras through a NLP task, deciding whether two questions in the quora dataset are duplicates or not.

There are 3 main notebooks in this:
- Data preprocessing
- Indexing the data
- Padding, and then actually training

I think these notebooks are still rough around the edges, so I'll be adding some more narrative commentary (as opposed to just code comments). I've also tried to make it work with python 2 and python 3, but I haven't tested it with 2 just yet.

To Do:
- [x] Write a sort of "next steps" guide for things people should try to make the model better. off the top of my head: pretrained embeddings (this one is important and likely to yield big gains), changing the BiLSTM output dimension, trying to use a BiGRU, trying to share weights between the question encoders / using the same encoder to encode both questions, trying different similarity functions, trying an entirely different architecture.

- [x] Write a readme with instructions on how to get the data

- [ ] Make some slides that give some context to all of this.

### CODE REVIEW

**Comment by nelson-liu** [comment]
requested a review from @ohkhan , but I think @ryanai3 (mostly bc you're familiar with keras, and this is a quick stab at something we've been meaning to try for awhile) and @leejt would also be interested in how this is shaping up?

**Comment by nelson-liu** [comment]
here's a link to the additions on my fork, since the diff is unreadable w/ ipynbs: https://github.com/nelson-liu/lectures/tree/keras_nlp_tutorial/2016-2017.Meetings/spring/01.keras_tutorial_duplicate_questions

**Review by ohkhan — APPROVED** [review]

**Comment by nelson-liu** [comment]
Merging, thanks for taking a look @ohkhan
